### DISCUSSION TITLE
Only email notifications for work orgnization; only GitHub UI notifications for non-work organization

### DISCUSSION THREAD

**Discussion by kleinfreund** [community/community#40141] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

*(I'm opening this discussion in the profile section because there isn't one for settings and none of the other categories seem more appropriate to me than "profile")*

I use one GitHub account for personal development and work.

**What I want**:

I'd like to control what *type* of notification (i.e. "On GitHub", "Email") to get *per custom notification route* (i.e. where you can already control to which email notifications are being routed based on organization, see https://github.com/settings/notifications/custom_routing).

On https://github.com/settings/notifications/custom_routing, this would allow me to set-up notifications as such:

* **Organization 1** (e.g. personal org, OSS project org):
  * Types of notification: On GitHub
  * Email: personal email address
* **Organization 2** (e.g. work org):
  * Types of notification: Email
  * Email: work email address

**Why I want it**:

* I currently can't turn off "On GitHub" notifications _for only work stuff_. That's annoying because I have to manage them twice: once in my work email; once in the GitHub UI.
* I currently can't avoid seeing work-related notifications in the GitHub UI when I'm off work. I _can_ avoid checking my work email, but I _can't_ avoid checking my GitHub UI notifications because they might have relevant stuff for my personal projects and contributions.

The only workaround for this right now is to disable GitHub UI notifications and only use email (or vice-versa). But I don't want that. I should be able to control routing to this level of detail.

**Comment by ossanna16** [comment]
Hi there @kleinfreund and welcome to our community! Thank you for the feedback!
